loopy/transform/iname.py

```diff
@@ -1639,11 +1639,15 @@ def map_reduction(self, expr, expn_state):
 
 
 @for_each_kernel
-def make_reduction_inames_unique(kernel, inames=None, within=None):
+def make_reduction_inames_unique(kernel, inames=None, within=None,
+                                 do_not_propagate_metadata=False):
```

Comment:
If you do decide to keep the flag, maybe reverse the sense: `propagate_tags=True`?

Reply:
Yep, got rid of the flag. Thanks!